charts/gateway/grafana-dashboards/gateway.json

```diff
@@ -3877,16 +3877,16 @@
           "type": "prometheus",
           "uid": "${DS_PROMETHEUS}"
         },
-        "definition": "label_values(gateway_latency_request_response_seconds{job=\"$job\", pod=~\"$pod\"},tenant)",
+        "definition": "label_values(vcluster)",
         "hide": 0,
         "includeAll": true,
-        "label": "Tenant",
+        "label": "vCluster",
         "multi": true,
         "multiFormat": "glob",
-        "name": "tenant",
+        "name": "vcluster",
         "options": [],
         "query": {
-          "query": "label_values(gateway_latency_request_response_seconds{job=\"$job\", pod=~\"$pod\"},tenant)",
+          "query": "label_values(vcluster)",
```

Comment:
Are you sure about this one ? vcluster is a label linked to metric, not a metric in itself

Reply:
Yes, this retrieve the list as expected: 
<img width="570" alt="Screenshot 2024-06-12 at 12 07 43" src="https://github.com/conduktor/conduktor-public-charts/assets/112936799/4af99a40-f95b-4d9d-95f1-2c5bea3f3060">